openfl/component/director/director.py

```diff
@@ -264,7 +271,7 @@ async def start_experiment_execution_loop(self):
         while True:
             async with self.experiments_registry.get_next_experiment() as experiment:
                 loop = asyncio.get_event_loop()
-                run_aggregator_future = loop.create_task(experiment.start(
+                self.run_aggregator_future = loop.create_task(experiment.start(
```

Comment:
`run_aggregator_future` - used only in this function

Reply:
Changed